.eslintrc.js

```diff
@@ -57,7 +57,7 @@ module.exports = {
       // 'jest'
     ],
     rules: {
-      'comma-dangle': 0,
+      'comma-dangle': 2,
```

Comment:
This rule can actually probably be entirely removed (rather than enabled or disabled) as ash-nazg includes the standard config which has this rule set to an even stricter setting (preventing dangling commas in functions as well, since that is allowed as of ES2017). (FWIW, I started some work on getting eslint-config-ash-nazg to automatically list the rules it is inheriting so it will be easier to search or browse the repo to see which rules--such as comma-dangle--are inherited "as is" from other configs.)

Reply:
Done!